react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+
+
+import { connect } from 'react-redux';
+import { arrayOf } from 'prop-types';
+
+import actionsCreators from '../../../../../redux/table-matches/actions';
+
+import styles from './styles.module.scss';
+
+var Spinner = require('react-spinkit');
+
+class TableMatches extends Component {
+  componentDidMount() {
+    this.props.getMatches();
+  }
+
+  renderMatch(match) {
```

Comment:
This looks more like a component than a class method. What do you think? Can we use it as a React component?
Also, you can destructure `match` to avoid repeating `match. `